src/lib/Timeline.js

```diff
@@ -425,11 +425,12 @@ export default class ReactCalendarTimeline extends Component {
         (this.state.visibleTimeStart - this.state.canvasTimeStart)) /
         newZoom
     )
-    if (this.scrollComponent.scrollLeft !== scrollLeft) {
+    const componentScrollLeft = Math.round(this.scrollComponent.scrollLeft);
+    if (componentScrollLeft !== scrollLeft) {
       this.scrollComponent.scrollLeft = scrollLeft
     }
 
-    if (this.scrollHeaderRef.scrollLeft !== scrollLeft) {
+    if (componentScrollLeft !== scrollLeft) {
```

Comment:
Thanks a lot for this PR, I've been having the same issues!

If we are doing the same conditional if check here `componentScrollLeft !== scrollLeft`, we could also move `this.scrollHeaderRef.scrollLeft = scrollLeft` into the previous `if` block :-)

Reply:
Yeah! Done! @tobiasbueschel